src/main/telemetry/msp_shared.h

```diff
@@ -20,13 +20,11 @@
 
 #pragma once
 
+#include "telemetry/crsf.h"
```

Comment:
It seems very wrong that MSP telemetry has a dependency on crossfire telemetry.

Reply:
fixed